### ISSUE TITLE
Configuration support for VelocityUtils

### ISSUE THREAD

**Issue by DoNotSpamPls** [DoNotSpamPls/VelocityUtils#2]
I don't have an idea how to accomplish this. Ideally it should be implemented with configurate-yaml and load the config from /src/main/resources/

Here is some code I've been trying, hope it may be useful to you:
```
private final Path configPath;
public static ConfigurationNode node;

public VelocityUtils(ProxyServer server, Logger logger, @DataDirectory Path configPath) {
    this.server = server;
    this.logger = logger;
    this.configPath = configPath;
}

private void loadConfig() {
    configPath.toFile().mkdir();
    InputStream configUrl = VelocityUtils.class.getResourceAsStream("/resources/config.yml");
    Path config = Paths.get("plugins", "velocityutils", "config.yml");
    YAMLConfigurationLoader loader = YAMLConfigurationLoader.builder().setPath(config).build();
    try {
        Files.copy(configUrl, config);
        node = loader.load();
    } catch (IOException ignored) {}
}
```

**Comment by satishthulva** [comment]
Please let us know what are the things that need to be configured ?

**Comment by DoNotSpamPls** [comment]
@satishthulva Hey there, thanks for your interest in this! Here what I'd like to be configurable about the alert command - feel free to make only the initial implementation, and I'll fill it with the configuration values I need later.

```
alert:
  prefix: "&8[&4Alert&8] &r"
  nopermission: "&4You do not have permission to execute this command!"

find:
  nopermission: "&4You do not have permission to execute this command!"
```

**Comment by DoNotSpamPls** [comment]
#3 adds a foundation for this, will close this issue once the configuration is fully done on my side.

**Comment by DoNotSpamPls** [comment]
Implemented in 4e7f842 (version 1.1.0). Thanks @Bennyboy1695 and @bhopahk
